docs-chef-io/content/inspec/waivers.md

```diff
@@ -40,7 +40,8 @@ control_id:
   justification: "reason for waiving this control"
 ```
 
-- `expiration_date` is optional. Absence means the waiver is permanent.
+- `expiration_date` is optional. Absence means the waiver is permanent. If the expiration date is `2019-10-15` then
+  it expires at `00:00` on the 15th of October. This considers the system date and time on which you are executing the inspec command to verify the expiration date.
```

Comment:
Is this 15th of October UTC or an offset based on the computer it's running on?

Reply:
The offset based on the computer it's running on.

Reply:
Cool, can we be explicit about that please?

It's a tad confusing :)